marklogic-data-hub-central/ui/src/pages/Browse.test.tsx

```diff
@@ -5,12 +5,36 @@ import {MemoryRouter} from "react-router-dom";
 import Browse from "./Browse";
 import {SearchContext} from "../util/search-context";
 import userEvent from "@testing-library/user-event";
+import axiosMock from "axios";
+import {act} from "react-dom/test-utils";
 
 jest.mock("axios");
 jest.setTimeout(30000);
 
-
 describe("Explorer Browse page tests ", () => {
+
+  const modelsResponse = [
```

Comment:
Looks like we are already storing mock entity model responses and sample hub central config responses in few files under `src/assets/mock-data/modeling/modeling.js`  and `src/assets/mock-data/explore/model-response.js`.
Can we place this mock data as well at one of those appropriate places and refer them here, as needed. ?